homework/english/RudayV/Lecture1/grammar.md

```diff
@@ -0,0 +1,47 @@
+##Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Your mum working today?
+Why this boys are talking now?
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she does not to drive a Mercedes.
+Let´s go out. It is not raining now.
+I have to go now. It is geting dark
+A young man usually stand at the window.
```

Comment:
A young man usually **stands** at the window.

Reply:
corrected mistakes